app/controllers/admin_masters_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+class AdminMastersController < ApplicationController
+  before_action :set_admin_master, only: %i[show edit update destroy]
+
+  # GET /admin_masters or /admin_masters.json
+  def index
+    @admin_masters = AdminMaster.all
+  end
+
+  # GET /admin_masters/1 or /admin_masters/1.json
+  def show; end
+
+  # GET /admin_masters/new
+  def new
+    @admin_master = AdminMaster.new
+  end
+
+  # GET /admin_masters/1/edit
+  def edit; end
+
+  # POST /admin_masters or /admin_masters.json
+  def create
+    @admin_master = AdminMaster.new(admin_master_params)
+
+    respond_to do |format|
+      if @admin_master.save
+        format.html { redirect_to admin_master_url(@admin_master), notice: "Admin master was successfully created." }
+        format.json { render :show, status: :created, location: @admin_master }
```

Comment:
do you need a json response overall? You can remove it in next PR if you are not planning to implement any API

Reply:
wil be update in th next mr